### ISSUE TITLE
conflict with python 3.9

### ISSUE THREAD

**Issue by awr-paul** [conda-forge/gdb-feedstock#41]
The gdb conda-forge package does not appear to install into a python 3.9 conda environment.  Please advise...

```
$ conda create --name gdb_test python=3.9
$ conda activate gdb_test
$ conda install gdb

Solving environment: / 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed                                                                          

UnsatisfiableError: The following specifications were found
to be incompatible with the existing python installation in your environment:

Specifications:

  - gdb -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0|>=3.5,<3.6.0a0|3.4.*']

Your python: python=3.9

If python is on the left-most side of the chain, that's the version you've asked for.
When python appears to the right, that indicates that the thing on the left is somehow
not available for the python version you are constrained to. Note that conda will not
change your python version to a different minor version unless you explicitly specify
that.
```

Many thanks,

Paul

**Comment by phil-blain** [comment]
Hi Paul, Python 3.9 support in GDB was only released in GDB 10.1 (see https://github.com/conda-forge/gdb-feedstock/pull/35#issuecomment-708120953) and I haven't been able to build GDB 10.1 in the conda-forge environment (see #38). You are welcome to help if it is something you need.

**Comment by phil-blain** [comment]
GDB 10.2 was added to the feedstock in #43, see https://anaconda.org/conda-forge/gdb/files?version=10.2. This includes a build with Python 3.9. Feel free to give it a go. I'll close this, please re-open  if you have further issues.

**Comment by blackbox-tech** [comment]
Fantastic, I'll give it ago.

**Comment by awr-paul** [comment]
Great, I'll try it out, thanks for letting me know.

> Hi Paul, Python 3.9 support in GDB was only released in GDB 10.1 (see [#35 (comment)](https://github.com/conda-forge/gdb-feedstock/pull/35#issuecomment-708120953)) and I haven't been able to build GDB 10.1 in the conda-forge environment (see #38). You are welcome to help if it is something you need.
